transitions/multi_form_alerts.js

```diff
@@ -153,22 +142,45 @@ const validateConfig = (alert) => {
   }
 };
 
+const count = (alert, latestReport) => {
```

Comment:
I think it would be helpful to either write a short comment explaining what this function does, or rename it to something less generic than `count`